js.es5.md

```diff
@@ -1,6 +1,6 @@
-# K&L JavaScript Style Guide
+# Education ES5 JavaScript Style Guide
```

Comment:
As discussed, will keep the 2 separate files (following AirBnB's example) - keeping the ES5 guide for reference for updating old codebases.